openslides/topics/models.py

```diff
@@ -17,19 +18,26 @@ class Topic(RESTModelMixin, models.Model):
     text = models.TextField(blank=True)
     attachments = models.ManyToManyField(Mediafile, blank=True)
 
+    # In theory there could be one then more agenda_item. But support only one.
+    # See the property agenda_item.
+    agenda_items = GenericRelation(Item)
+
     class Meta:
         default_permissions = ()
 
     def __str__(self):
         return self.title
 
+    @classmethod
+    def all_query(cls):
+        return cls.objects.prefetch_related('attachments', 'agenda_items')
+
```

Comment:
[This](https://docs.djangoproject.com/en/1.10/topics/db/managers/#custom-managers-and-model-inheritance) does not work? I would prefer not to override the default manager `objects` but to add a custom manager `autoupdate_objects` that uses the `get_queryset()` method. So we still have access to the default methods with default Django behavior where we do not want to prefetch some stuff (e. g. calling Model.objects.all() with only one query).